armada-ths/ais
# Student matching questions

Implemented initial student matching questions model and api

At the moment there are only slider questions, but adding extra types will take less effort, due to the way I designed it.

Slider questions can be created from the admin view (in Matching app) and selecting wrong type there doesn't mean anything, as it will be overriden.

To test:
* run migrations
* run tests
* goto localhost:8000/admin and create at least one slider question
* goto localhost:8000/api/questions/ and make sure you get a json file {"questions":[{set of questions}]}


++ by Viktor:t the full matching db should be setup, not much to test except going through the matching/tests.py and make sure they make sense (I am testing all the db relations there)

The result model in student_profiles/models. is also implemented with a corresponding test in student_profiles/tests.py
